src/test/addresses_goerli.sol

```diff
@@ -298,6 +299,18 @@ contract Addresses {
         addr["RWA006_A_URN"]                    = 0xd0d2Ef46b64C07b5Ce4f2634a82984C1B3804C22;
         addr["RWA006_A_INPUT_CONDUIT"]          = 0xd2Ef07535267D17d2314894f7821A43e9700A02e;
         addr["RWA006_A_OUTPUT_CONDUIT"]         = 0xd2Ef07535267D17d2314894f7821A43e9700A02e;
+        addr["RWA008"]                          = 0x30434AA15F85598F45406d08A79dCdD9a79335e9;
+        addr["PIP_RWA008"]                      = 0x98e62fFAf27C022283cB492f1bB05AfdE877b5ac;
```

Comment:
Did you have to simulate what the `init()` would produce to get this address and the `PIP_RWA009` address?

Reply:
Yes. This works well if we don't make other calls to `init()` in the mean time.

Reply:
Solving this.